Dnd.Ddd.Services/Commands/Handlers/ChooseCharacterNameCommandHandler.cs

```diff
@@ -34,10 +34,10 @@ public void Handle(ChooseCharacterNameCommand command)
 
             Guard.With<InvalidOperationException>()
                 .Against(
-                    !(character is CharacterDraft),
+                    character.IsCompleted(),
                     $"Attempting to change name on a completed character with UiD: {command.CharacterUiD}!");
```

Comment:
Given that we introduced ```InvalidCharacterStateException```, I think we could use it here (add necessary using beforehand):
```suggestion
Guard.With<InvalidCharacterStateException>().Against(character.IsCompleted(), character.UiD);
```

Reply:
changed